docs/multiple-base-images.md

```diff
@@ -25,3 +28,61 @@ reasons to put everything in one image than to separate them:
 - If you have a step which is really lightweight, there's a better option than having
   a small new container: _no_ new container. This is a great case for inline functions.
   See the [execution mode docs](https://docs.sematic.dev/execution-modes) for more.
+
+### Multiple images if you must
+
+There are cases where step-specific images can be warranted. For example one
+step uses a third-party library that comes packaged as a container image.
+
+For this reason, Sematic enables specifying multiple **base** images. This means
+that Sematic will use these different images as bases on top of which your
+pipeline code will be packaged to make sure the necessary code and dependencies
+are available at runtime. Sematic will also override the entry point of the
+image with the standard Sematic worker entry point.
+
+The API to specify a **single** base image for all your pipeline is as follows in your pipeline's Bazel `BUILD` file:
+
+```starlark
+load("@rules_sematic//:pipeline.bzl", "sematic_pipeline")
+
+sematic_pipeline(
+  name="main",
+  registry = "your.image.registry",
+  repository = "image-repo",
+  base = "@your-image-target",
+  deps = [...],
+)
+```
+
+To specify different base images for different steps remove the `base` argument and instead pass a mapping to the `bases` argument:
+
+```starlark
+sematic_pipeline(
+  name="main",
+  ...
+  bases={
+    "cuda": "@cuda-base-target"
+  }
+)
+```
+Then in your pipeline code, specify which image to use by passing the
+corresponding key to the Sematic decorator:
+
+```python
+@sematic.func(base_image_tag="cuda")
+def train_model(...):
+  ...
+```
+
+Then, an image built on top of `@cuda-base-target` is used to execute this particular function.
+
+Notes:
+
+- Nested functions (i.e. if `train_model` calls other Sematic functions), will
+  execute according to their own settings, i.e. with the default image unless
+  otherwise specified.
+- A default base image to be used for all steps where no `base_image_tag` is
+  passed can be specified either by setting the `base` argument like in the
+  single image case above, or setting an image target with the `"default"` key
```

Comment:
The instructions above say to remove "base" completely. Please reword line 57 and instead say to add "basses" in addition to "base".